build-and-test.ps1

```diff
@@ -10,28 +10,26 @@ $ErrorActionPreference = 'Stop'
 
 $buildFilter = "$VersionFilter-$OSFilter/*"
 
-function Get-ImageBuilder()
-{
-    $imageBuilderImageName = 'microsoft/dotnet-buildtools-prereqs:image-builder-nanoserver-20180105103709'
-    $imageBuilderContainerName = "ImageBuilder-$(Get-Date -Format yyyyMMddhhmmss)"
-    New-Item -Path "$imageBuilderFolder" -ItemType Directory -Force | Out-Null
+function ExecuteCommand {
```

Comment:
I think it would be helpful if the name called out that that retry logic exists.  This helps clarify why the function is used over Invoke-Expression.  e.g. ExecuteWithRetry

Reply:
I did consider the name `ExecuteWithRetry`. However, with the word `Retry` the questions that come to my (user's) mind is - how do I set the number of retries, wait time? Is the wait time in seconds or minutes, etc.

Reply:
Since this is a local method, I think it is reasonable for the reader to look at the implementation.  I think `ExecuteWithRetry` addresses the point I raised on why the method exists over using the built in Invoke-Expression and why this method is only being used for the pull cmd.